validator.py

```diff
@@ -403,7 +403,7 @@ def get_servers(region: str, token: str):
     else:
         # Fetch the servers and assets from the API
         asset_tags = list(set(server['asset_tag'] for server in servers if server['asset_tag'] is not None))
-        asset_params = {'assetTag__in': asset_tags, 'idAddress': region}
+        asset_params = {'asset_tag__in': asset_tags, 'address_id': region}
```

Comment:
```suggestion
        asset_params = {'assetTag__in': asset_tags, 'idAddress': region}
        # Asset Application upgrade to Python 3 is in dev, please uncomment below
        # and comment out above to test in dev. 
        # asset_params = {'asset_tag__in': asset_tags, 'address_id': region}
```

Should we leave a comment like this so validator can also be executed in production environments?